backend/tests/integration/files/test_api_s3.py

```diff
@@ -119,6 +119,18 @@ def test_get_file_url(base_app, appctx):
 
 
 def test_get_content_disposition_with_broken_filename(base_app, appctx):
-    expected_content_disposition = 'attachment; filename="Mass_mm_.s"'
-    result = current_s3_instance.get_content_disposition("Mass_mm\n.s\n")
+    expected_content_disposition = 'inline; filename="Mass_mm_.s"'
+    result = current_s3_instance.get_content_disposition("Mass_mm\n.s;b\n")
+    assert result == expected_content_disposition
+
+
+def test_get_content_disposition_with_subformat(base_app, appctx):
+    expected_content_disposition = 'inline; filename="filefulltext.pdf"'
```

Comment:
is this what we want? with ```secure_filename```, it would have still had the ```.``` between full and text and you are removing it. I think it's fine but I just wanted to put the question it out there.

Reply:
(this is a behaviour from before) `werkzeug` thinks they are insecure, and I don't think we're losing anything useful there anyway.

Reply:
Actually, that's what I'm saying, ```werkzeug``` was keeping the ```.``` before so it isn't a behavior from before. ```secure_filename('full.text.pdf')``` would be ```full.text.pdf'``` whereas now you are removing the ```.```. I don't think it is relevant though, I don't know what weird filenames we have

Reply:
true, you're right I forgot with `"."` back.. it's probably not important but better to keep.